content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-beta/index.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: blog
+title: 'Kubernetes 1.33: Fine-grained SupplementalGroups control graduates to beta'
+date: 2025-04-23
+draft: true
+slug: fine-grained-supplementalgroups-control-beta
+author: >
+  Shingo Omura (LY Corporation)
+
+---
+
+Pod supplemental groups policy control for Pods is now beta. Read on to learn how it will help your cluster security, and how to start using it.
+ 
+The new field was introduced as an opt-in alpha feature for Kubernetes 1.31 and has graduated to beta in 1.33; the corresponding feature gate (`SupplementalGroupsPolicy`) is now enabled by default.
```

Comment:
Introduce the field detail early to let the readers know what this blog is about

```suggestion
The new field, `supplementalGroupsPolicy`, was introduced as an opt-in alpha feature for Kubernetes 1.31 and has graduated to beta in 1.33; the corresponding feature gate (`SupplementalGroupsPolicy`) is now enabled by default.
```